dist/gaiden.min.css

```diff
@@ -1,2 +1,14 @@
splay:-webkit-box;display:-webkit-flex;display:-moz-box;display:-ms-flexbox;display:flex}.media__image--mini{max-height:175px}.media__title{color:#536e7b}.media__description{display:inline-table}.media__content{color:rgba(0,0,0,0.6)}.container{-webkit-box-direction:normal;-webkit-box-orient:horizontal;-webkit-flex-direction:row;-moz-flex-direction:row;-ms-flex-direction:row;flex-direction:row;-webkit-flex-wrap:wrap;-moz-flex-wrap:wrap;-ms-flex-wrap:wrap;flex-wrap:wrap;display:-webkit-box;display:-webkit-flex;display:-moz-flex;display:-ms-flexbox;display:-moz-box;display:flex;margin-right:auto;margin-left:auto;padding-right:1.25%;padding-left:1.25%;width:100%}.no-flexbox .container::before,.no-flexbox .container::after{display:table;content:''}.no-flexbox .container::after{clear:both}.no-flexbox .container [class*='col-']{float:left}.container--fluid{width:100%;max-width:inherit}.container--middle-items{-webkit-box-align:center;-webkit-align-items:center;-moz-box-align:center;-ms-flex-align:center;align-items:center}[class*='col-']{-webkit-flex-shrink:0;-moz-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;-webkit-flex-wrap:wrap;-moz-flex-wrap:wrap;-ms-flex-wrap:wrap;flex-wrap:wrap;display:-webkit-box;display:-webkit-flex;display:-moz-flex;display:-ms-flexbox;display:-moz-box;display:flex;max-width:100%}[class*='col-'].col-centered{margin-right:auto;margin-left:auto}.container{max-width:420px}[class*='col-small-']{margin-right:1.25%;margin-left:1.25%}.col-small-1{width:5.83333%}.col-push-small-1{margin-left:9.58333%}.col-offset-small-1{left:8.33333%}.col-small-2{width:14.16667%}.col-push-small-2{margin-left:17.91667%}.col-offset-small-2{left:16.66667%}.col-small-3{width:22.5%}.col-push-small-3{margin-left:26.25%}.col-offset-small-3{left:25%}.col-small-4{width:30.83333%}.col-push-small-4{margin-left:34.58333%}.col-offset-small-4{left:33.33333%}.col-small-5{width:39.16667%}.col-push-small-5{margin-left:42.91667%}.col-offset-small-5{left:41.66667%}.col-small-6{width:47.5%}.col-push-small-6{margin-left:51.25%}.col-offset-small-6{left:50%}.col-small-7{width:55.83333%}.col-push-small-7{margin-left:59.58333%}.col-offset-small-7{left:58.33333%}.col-small-8{width:64.16667%}.col-push-small-8{margin-left:67.91667%}.col-offset-small-8{left:66.66667%}.col-small-9{width:72.5%}.col-push-small-9{margin-left:76.25%}.col-offset-small-9{left:75%}.col-small-10{width:80.83333%}.col-push-small-10{margin-left:84.58333%}.col-offset-small-10{left:83.33333%}.col-small-11{width:89.16667%}.col-push-small-11{margin-left:92.91667%}.col-offset-small-11{left:91.66667%}.col-small-12{width:97.5%}.col-push-small-12{margin-left:101.25%}.col-offset-small-12{left:100%}.space-section-small{margin-top:60px}.space-section-medium{margin-top:72px}.space-section-large{margin-top:80px}.space-box-small{margin-top:24px}.space-box-medium{margin-top:36px}.space-box-large{margin-top:48px}.space-element-small{margin-top:6px}.space-element-medium{margin-top:12px}.space-element-large{margin-top:16px}
```

Comment:
@iondrimba I think that not. Dist files is used as a official version of Gaiden and will be used on our document page (on `gh-pages` branch).